sdk/ai/ai-projects/README.md

```diff
@@ -439,6 +471,14 @@ console.log(`Created agent, agent ID : ${agent.id}`);
 const thread = await client.agents.createThread({ toolResources: fileSearchTool.resources });
 ```
 
+#### List Threads
+
+To list all threads attached to a given agent, use the list_threads API:
+
+```ts snippet:listThreads
+const threads = client.agents.listThreads();
```

Comment:
If this is an async function or an iterator I recommend showing the entire usage example (either with `await` or with `for await`)